openedx_blockstore_relay/tests/test_transfer_data.py

```diff
@@ -9,125 +9,108 @@
-
-        self.compat = mock.patch('openedx_blockstore_relay.transfer_data.compat', StubCompat({
-            self.block_key: self.block
-        }, assets=[
-            { 'content':StaticContent(''), 'path':'/static/one.jpeg'},
-            { 'content':StaticContent('<script></script>'), 'path':'/static/one.js'},
-        ], video_assets=[
-            {'content':StaticContent('1. Hello welcome to this course.'), 'path': '/static/transcript.srt'},
-        ]))
-        self.compat.start()
-
-
-    def test_bundle_creation(self):
-
-        with mock.patch('openedx_blockstore_relay.transfer_data.requests') as mock_requests:
-            with mock.patch('openedx_blockstore_relay.transfer_data.TransferBlock.transfer_block_to_bundle'):
-                transfer_to_blockstore(self.block_key, collection_uuid=self.COLLECTION_UUID)
-
-    def test_transfer_to_blockstore(self):
-
-        with mock.patch('openedx_blockstore_relay.transfer_data.requests') as mock_requests:
-
-            mock_response = mock.MagicMock()
-
-            # TODO: Remove once _add_bundle_file() adds the path to the manifest without looking at the response.
-            mock_response.json.return_value = {
-                'path': '/vertical/v1.olx',
-            }
-            mock_requests.post.return_value = mock_response
-
-            transfer_obj = TransferBlock(self.block_key)
-            transfer_obj.transfer_block_to_bundle(self.BUNDLE_UUID)
-
-            files_posted = set([call_args[1]['data']['path'] for call_args in mock_requests.post.call_args_list])
-
-            self.assertSetEqual(files_posted, {
-                '/bundle.json',
-                '/vertical/v1.olx',
-                '/video/video1.olx',
-                '/html/html1.olx',
-                '/static/one.jpeg',
-                '/static/one.js',
-                '/static/transcript.srt',
-            })
-
-            self.assertDictEqual(transfer_obj.manifest, {
-                'assets': [
-                    '/static/one.jpeg',
-                    '/static/one.js',
-                    '/static/one.jpeg',
-                    '/static/one.js',
-                    '/static/transcript.srt',
-                    '/static/one.jpeg',
-                    '/static/one.js'
-                ],
-                'components': ['/vertical/v1.olx'],
-                'dependencies': [],
-                'schema': 0.1,
-                'type': u'olx/unit',
-            })
+        self.course = SampleCourseFactory.create(block_info_tree=TEST_COURSE)
+        # And upload the course static asssets:
+        asset_key = StaticContent.compute_location(self.course.id, 'sample_handout.txt')
+        content = StaticContent(asset_key, "Fake asset", "application/text", "test".encode('utf8'))
+        contentstore().save(content)
+        # And the video data + transcript must also be stored in edx-val for the video export to work:
+        edxval_api.create_video(VIDEO_B_VAL_DATA)
```

Comment:
If we export a course from edx.org and import it into stage.labxchange.com edx instance, which will not have VEDA configured, will this still work?

Reply:
I was under the impression that edx-val was enabled by default from Hawthorn onward?

There is no feature flag for edx-val that I can see, it's simply [enabled if installed](https://github.com/edx/edx-platform/blob/0cfda2d09782ba0ddb3ee2632e5101ad6491f9f1/common/lib/xmodule/xmodule/video_module/video_module.py#L84-L87), and [it's installed by default](https://github.com/edx/edx-platform/blob/66b1c07b7195f9ffd9b9e211384465bce2294d3b/requirements/edx/base.in#L86)

Reply:
Oh okay, that would be great.